tpu-client-next/src/connection_workers_scheduler.rs

```diff
@@ -90,6 +91,65 @@ pub struct ConnectionWorkersSchedulerConfig {
     pub leaders_fanout: Fanout,
 }
 
+/// The [`WorkersBroadcaster`] trait defines a customizable mechanism for
```

Comment:
I'm thinking if this Broadcasters story should be move to the end of this file so that user first sees the most important structure of this component, what do you think? @ilya-bobyr

Reply:
As it is quite short and is part of the API, I think it is fine.

I would probably keep the trait here, but move the `NonblockingBroadcaster` to the end.  As it is not part par the API of the `ConnectionWorkersScheduler` but is a separate entity.